spec/ndx-sound.extensions.yaml

```diff
@@ -1,4 +1,100 @@
 groups:
+- neurodata_type_def: Speaker
+  neurodata_type_inc: Device
+  doc: Speaker device used for acoustic stimuli
+  datasets:
+  - name: frequency_range_in_hz
+    dtype: float
+    shape:
+    - 2
+    doc: Frequency range of the speaker in Hz
+  - name: sensitivity_in_db
+    dtype: float
+    doc: Sensitivity of the speaker in dB at 1m
+- neurodata_type_def: Microphone
+  neurodata_type_inc: Device
+  doc: Microphone device used for acoustic recordings
+  datasets:
+  - name: frequency_range_in_hz
+    dtype: float
+    shape:
+    - 2
+    doc: Frequency range of the microphone in Hz
+  - name: sensitivity_in_mv_per_pa
+    dtype: float
+    doc: Sensitivity of the microphone in mV/Pa
+- neurodata_type_def: AudioInterface
```

Comment:
What's an example device for this? : O 
Should it be `AudioInterfaceDevice` to avoid confusion with the concepts of interfaces in pynwb and the concept of interfaces in neuroconv?

Reply:
ah, of course, this:
https://rme-audio.de/fireface-ufx-ii.html